tests/lowest_fee.rs

```diff
@@ -158,6 +158,5 @@ fn combined_changeless_metric() {
         common::bnb_search(&mut cs_b, metric_combined, usize::MAX).expect("must find solution");
     println!("score={:?} rounds={}", combined_score, combined_rounds);
 
-    // [todo] shouldn't rounds be less since we are only considering changeless branches?
-    assert!(combined_rounds <= rounds);
+    assert!(combined_rounds >= rounds);
```

Comment:
@LLFourn mentioned in a private discussion:

> ... in theory the combined will be looser than each individual one in the sense that it will take more iterations to find the optimal solution for the combined one I think. (A,B) has to find an optimal solution for A but this will not necessarily be optimal for (A,B) so it has to keep looking.

I responded with:

> Would you agree that combining another metric with `Changeless` is only useful if it tightens the bounds? Otherwise, we might as will do `Iterator::filter` on the `BnbIter`.

I will create an issue for this discussion.